src/allencell_ml_segmenter/utils/cyto_overrides_manager.py

```diff
@@ -62,6 +62,9 @@ def get_training_overrides(
                 )
             # Filters/Model Size (required if starting new model)
             overrides_dict["model._aux.filters"] = model_size.value
+            # needed to specify no weights used
+            overrides_dict["checkpoint.weights_only"] = False
```

Comment:
how does `checkpoint.weights_only` get overidden?

Reply:
was that already implemented ?

Reply:
Does the config itself have a default?  It seems like if we want to default to False , that would be the ideal place to do it.

Reply:
Looks like that happens here: [in the overrides manager class](https://github.com/AllenCell/allencell-segmenter-ml/blob/5676077d5f81e3c519e18bcea9bdf5c98dfd8938/src/allencell_ml_segmenter/utils/cyto_overrides_manager.py#L42)

Reply:
Is there test cov for `weights_only` ?

Reply:
in `test_get_training_overrides_iterative_training` i see as asserts True on `weights_only` but i dont see anything setting `is_using_existing_model` in the ARRANGE section.

Reply:
`checkpoint.weights_only` should be True for iterative training runs and False for regular training runs.

The config's defaults are `checkpoint.weights_only = False` and `checkopint.strict = True` so we shouldnt need to explicitly set here, just wanted to ensure it does get set for non iterative training runs. I will remove this.

Reply:
> in `test_get_training_overrides_iterative_training` i see as asserts True on `weights_only` but i dont see anything setting `is_using_existing_model` in the ARRANGE section.

In this test we are setting `model.set_is_using_existing_model(True)` when arranging on line 117.

Reply:
Thanks for pointing out 117, i missed it .  lgtm.